backends/lwjgl3/src/main/java/com/anyicomplex/gdx/lwjgl3/svm/ReflectionRegistrationFeature.java

```diff
@@ -17,21 +17,15 @@
 
 package com.anyicomplex.gdx.lwjgl3.svm;
 
-import com.oracle.svm.core.annotate.AutomaticFeature;
 import org.graalvm.nativeimage.hosted.Feature;
 import org.graalvm.nativeimage.hosted.RuntimeReflection;
-import org.lwjgl.system.StructBuffer;
 
-@AutomaticFeature
 public class ReflectionRegistrationFeature implements Feature {
 
+
     @Override
     public void beforeAnalysis(BeforeAnalysisAccess access) {
 
-        boolean isWindows = System.getProperty("os.name").contains("Windows");
```

Comment:
At some point platform specific code will be probably needed to get added back, so I wanted to keep it for now

Reply:
OK, I'll add it back.